<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>128</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
   <item row="0" column="2">
    <widget class="QPushButton" name="separateBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Separate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="MyLineEdit" name="gamPath">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="gamLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GAM:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="libLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LIB:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="combineBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combine</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MyLineEdit" name="headPath">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="headLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HEAD:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="MyLineEdit" name="libPath">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="stateLbl">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
